src/oncall/api/v0/ical_key.py

```diff
@@ -10,6 +10,27 @@ def generate_ical_key():
     return str(uuid.uuid4())
 
 
+def check_ical_team(team, requester):
```

Comment:
I don't know if this function is something specific to the ical integration. It seems pretty reusable; after all, it's just a function that returns True if the team is a valid, active team. It might be better to move this to somewhere like the `auth` module?

Reply:
I agree that currently this function only check for a valid and active team, but I also passed in `requester` in case in the future we are restricting access based on users.

Also I am not sure if auth module is the best place for this function; maybe team related modules work better?

Reply:
Yeah that makes sense!

Reply:
Wait, so are you going to do this? Or defer this for a later commit?

Reply:
We can change this later